internal/openmetrics-exporter/common_tests.go

```diff
@@ -25,24 +24,3 @@ func metricsCheck(t *testing.T, c prometheus.Collector, want map[string]bool) {
 		t.Errorf("Mismatch (-want +got):\n%s", diff)
 	}
 }
-
-func metricsCheckP(t *testing.T, c prometheus.Collector, want map[string]bool) {
-	chM := make(chan prometheus.Metric)
-	go func() {
-		c.Collect(chM)
-		close(chM)
-	}()
-	var buff io_prometheus_client.Metric
-	metrics := make(map[string]bool)
-	for m := range chM {
-		m.Write(&buff)
-		metrics[buff.String()] = true
-	}
-        for a := range metrics {
-		fmt.Println(a)
-	}
-	fmt.Println("++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++")
-        for a := range want {
-		fmt.Println(a)
-	}
-}
```

Comment:
@genegr there was also this function that was not being used, please confirm its removal.